cpu/fe310/periph/rtt.c

```diff
@@ -161,8 +132,13 @@ void rtt_set_counter(uint32_t counter)
      * Must program HI/LO regs
      * Write scaled counter reg value
      */
-    AON_REG(AON_RTCLO) = counter << RTT_SCALE;
-    AON_REG(AON_RTCHI) = counter >> (32 - RTT_SCALE);
+#if RTT_CLOCK_FREQUENCY == RTT_FREQUENCY
+        AON_REG(AON_RTCLO) = counter;
+        AON_REG(AON_RTCHI) = 0;
```

Comment:
Not really needed, in this case you have `RTT_SCALE = 0`

```C
        AON_REG(AON_RTCLO) = counter << 0; // == counter
        AON_REG(AON_RTCHI) = counter >> (32 - RTT_SCALE); // == 0
```

Reply:
Did this because of the compiler, `AON_REG(AON_RTCHI) = counter >> (32 - 0);` is considered out of bound shift.

Reply:
I can add a comment

Reply:
Added a comment.

Reply:
Added comment.